styleguide/derek/examples/_office365.html

```diff
@@ -0,0 +1,88 @@
+<!--Header Large-->
+<div class="container-fluid banner large outlook">
+  <div class="row">
+    <div class="col-sm-5 center-block">
+      <div class="banner-text">
+        <h1 class="center white">Fanatical Support for Office 365</h1>
+        <p class="center white">All the power of Office 365. None of the frustration of migration and management. We help you get the most out of Office 365. </p>
+      </div>
+    </div>
+  </div>
+  <div class="row">
+      <div class="col-sm-4 center-block">
+          <button>Choose A Plan</button>
+      </div>
+  </div>
+</div>
+
+<nav class="navbar subnav">
```

Comment:
@amberRrucker sorry, I forgot to mention that you won't need this with the `_navbar` partial in play now. I am seeing double menus.